app/controllers/v2/versions_controller.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module V2
+  # API for delivering files from stacks by version
+  class VersionsController < ApplicationController
```

Comment:
I guess I don't understand why we need a new controller (that's nearly the same as `FileController`). That seems unfortunate.

Reply:
That was the suggestion in the ticket. Happy to revert (maybe can be a standup discussion or a follow up pr if this is blocking)

Reply:
I thought this was the easiest way to disambiguate the route. The current route allows slashes in filename, so adding a version there may have the potential of collision with a directory or filename.

Reply:
A separate route seems fine, I just don't know why we need a nearly identical controller to go with it.

Reply:
I can go either way on that.  One controller is fine by me.

Reply:
@jcoyne / @cbeer - I removed the second controller.